### DISCUSSION TITLE
Auto shown the application volume when it is changed with keybind in the notifications

### DISCUSSION THREAD

**Discussion by JavierL742** [radj307/volume-control#89] [Ideas & Feature Discussions]
Hello, i like this app a lot, and i appreciate your work. Thank you for what you are doing.

This is one feature i would like to see. Example photo:

![image](https://github.com/radj307/volume-control/assets/67005972/5486d966-f4ed-4269-b67d-8924e0b6c0b1)

Regards and thank you once more👋

**Comment by radj307** [comment]
Just need some clarification on what you mean when you say you have a `Ctrl + A` hotkey. Do you mean that the `Ctrl + A` hotkey has Discord as a target override?  
![target override window](https://github.com/radj307/volume-control/assets/1927798/3928a140-c3f8-46b4-819f-3c465c47db6b)  
If so, that should be pretty easy to implement.

**Reply by JavierL742** [reply]
Correct!

**Comment by radj307** [comment]
This should be no problem! I can easily add a checkbox to the action settings window to enable this behaviour. I've converted this discussion into issue #92 to track its progress.
